specification/conformance/conformance_rules/datasets/costandusage.json

```
@@ -0,0 +1,2116 @@
+        "CapacityReservationStatus-C-000-C",
+        "ChargeCategory-C-000-M",
+        "ChargeClass-C-000-M",
+        "ChargeFrequency-C-000-O",
+        "ChargeDescription-C-000-M",
+        "ChargePeriodEnd-C-000-M",
+        "ChargePeriodStart-C-000-M",
+        "CommitmentDiscountCategory-C-000-C",
+        "CommitmentDiscountId-C-000-C",
+        "CommitmentDiscountName-C-000-C",
+        "CommitmentDiscountStatus-C-000-C",
+        "CommitmentDiscountType-C-000-C",
+        "CommitmentDiscountUnit-C-000-C",
+        "CommitmentDiscountQuantity-C-000-C",
+        "ConsumedUnit-C-000-C",
+        "ConsumedQuantity-C-000-C",
+        "ContractedCost-C-000-M",
+        "ContractedUnitPrice-C-000-C",
+        "EffectiveCost-C-000-M",
+        "InvoiceId-C-000-O",
+        "InvoiceIssuerName-C-000-M",
+        "ListCost-C-000-M",
+        "ListUnitPrice-C-000-C",
+        "PricingCategory-C-000-M",
+        "PricingCurrency-C-000-C",
+        "PricingCurrencyContractedUnitPrice-C-000-C",
+        "PricingCurrencyEffectiveCost-C-000-C",
+        "PricingCurrencyListUnitPrice-C-000-C",
+        "PricingQuantity-C-000-M",
+        "PricingUnit-C-000-M",
+        "ProviderName-C-000-M",
+        "RegionId-C-000-C",
+        "RegionName-C-000-C",
+        "ResourceId-C-000-C",
+        "ResourceName-C-000-C",
+        "ResourceType-C-000-C",
+        "PublisherName-C-000-M",
+        "SubAccountId-C-000-C",
+        "SubAccountName-C-000-C",
+        "ServiceCategory-C-000-M",
+        "ServiceName-C-000-M",
+        "ServiceSubcategory-C-000-O",
+        "SkuId-C-000-C",
+        "SkuMeter-C-000-C",
+        "SkuPriceDetails-C-000-C",
+        "SkuPriceId-C-000-C",
+        "SubAccountType-C-000-C",
+        "Tags-C-000-C"
+      ]
+    }
+  },
+  "CostAndUsage-D-003-C": {
+    "Function": "Presence",
+    "Reference": "AvailabilityZone",
+    "EntityType": "Dataset",
+    "Notes": "",
+    "CRVersionIntroduced": "1.2",
+    "Status": "Active",
+    "ApplicabilityCriteria": [
+      "AVAILABILITY_ZONE_SUPPORTED"
+    ],
+    "Type": "Static",
+    "ValidationCriteria": {
+      "MustSatisfy": "AvailabilityZone is RECOMMENDED to be present in a FOCUS dataset when the provider supports deploying resources or services within an availability zone.",
+      "Keyword": "RECOMMENDED",
+      "Requirement": {
+        "CheckFunction": "ColumnPresent",
+        "ColumnName": "AvailabilityZone"
+      },
+      "Condition": {},
+      "Dependencies": []
+    }
+  },
+  "CostAndUsage-D-004-C": {
+    "Function": "Presence",
+    "Reference": "ListUnitPrice",
+    "EntityType": "Dataset",
+    "Notes": "",
+    "CRVersionIntroduced": "1.2",
+    "Status": "Active",
+    "ApplicabilityCriteria": [
+      "PUBLIC_PRICE_LIST_SUPPORTED"
+    ],
+    "Type": "Static",
+    "ValidationCriteria": {
+      "MustSatisfy": "ListUnitPrice MUST be present in a FOCUS dataset when the data generator publishes unit prices exclusive of discounts.",
```

The only case in costandusage.json where the condition phrasing doesn’t match the 1.2 spec (35/36 match). The same condition (but with `provider`) appears six more times.
Suggest updating for consistency:

```suggestion
      "MustSatisfy": "ListUnitPrice MUST be present in a FOCUS dataset when the provider publishes unit prices exclusive of discounts.",
```